insolar/insolar
# NOISSUE: fix pulsar distribution

**- What I did**
1. Fixed bug in pulsar distributor that caused timeouts
2. Now pulsar sends pulses only to the nodes from `pulsedistributor.bootstraphosts` configuration params (getRandomHosts is not called)
3. Added magic number `Skip` to servicenetwork configuration that allows to skip a couple of pulses before network bootstrap to have synchronized first pulse on all nodes (very helpful on short pulses < 3000ms)
4. Pulsar config in launchnet is now generated based on `pulsar_template.yaml` (all settings except bootstraphosts) and `genesis.yaml` (bootstraphosts)

**- How to verify it**
Run launchnet.sh on 20+ nodes and check with script `ag -m1 'persist' scripts/insolard/nodes/*/output.log 2>/dev/null | awk '{print $4}' | sort -u` that all nodes start from one pulse
